packages/@uppy/tus/types/index.d.ts

```diff
@@ -1,24 +1,16 @@
-import Uppy = require('@uppy/core');
+import Uppy = require('@uppy/core')
 
 declare module Tus {
   export interface TusOptions extends Uppy.PluginOptions {
```

Comment:
should we also extend type options from the `tus-js-client`? 🤔If it's available in its type declarations i.e

Reply:
hmm yeah we should, tus-js-client options minus the ones that we override unconditionally :+1: